main.py

```diff
@@ -72,10 +71,11 @@ def LJ(use_asap=use_asap):
 
     # Set the momenta corresponding to T=300K
     MaxwellBoltzmannDistribution(atoms, temperature_K=options["temperature_K"])
+    # Is this where the temperature is halfed??
     Stationary(atoms)
     ZeroRotation(atoms)
     # We want to run MD with constant energy using the VelocityVerlet algorithm.
-    dyn = VelocityVerlet(atoms, 5 * units.fs)  # 5 fs time step.
+    dyn = VelocityVerlet(atoms, 1 * units.fs)  # 5 fs time step.
```

Comment:
This should be read from config file, called dt

Reply:
I haven't touched on these and I don't think the merge from develop covered that, however I  can go and change it.